core/server/common/src/main/java/alluxio/master/journal/JournalSystem.java

```diff
@@ -189,7 +189,7 @@ public Builder setQuietTimeMs(long quietTimeMs) {
     /**
      * @return a journal system
      */
-    public JournalSystem build() {
+    public JournalSystem build(CommonUtils.ProcessType processType) {
```

Comment:
since we only use processType to determine serviceType, why not directly pass the serviceType?

Reply:
Because we are not necessarily building Embedded journal system (there are also noop journal system and ufs journal system), always passing in `ServiceType.MASTER_RAFT` and `ServiceType.JOB_MASTER_RAFT` feels like we are building embedded journal system.